modules/api/src/main/UserGameApi.scala

```diff
@@ -49,13 +50,17 @@ final class UserGameApi(
     }),
     "fen" -> Forsyth.exportBoard(g.board),
     "winner" -> g.winnerColor.map(_.name),
-    "bookmarks" -> g.bookmarks
-  ).add("bookmarked" -> bookmarked)
-    .add("analysed" -> g.metadata.analysed)
-    .add("opening" -> g.opening)
-    .add("lastMove" -> g.lastMoveKeys)
-    .add("clock" -> g.clock)
-    .add("correspondence" -> g.daysPerTurn.map { d =>
+    "bookmarks" -> g.bookmarks,
+    "bookmarked" -> bookmarked,
+    "analysed" -> g.metadata.analysed,
+    "opening" -> g.opening,
+    "lastMove" -> g.lastMoveKeys,
+    "clock" -> g.clock,
+    "correspondence" -> g.daysPerTurn.map { d =>
       Json.obj("daysPerTurn" -> d)
-    })
+    },
+    "tournament" -> g.tournamentId.map { tid =>
+      Json.obj("id" -> tid, "name" -> getTournamentName(tid))
+    }
```

Comment:
I like `.add` because it skips empty fields, and helps keep the JSON output as small as possible. Any reasons why you need `null`s in the JSON?

Reply:
ah ok, no reason. I just wanted to simplify the code and did not think about `null`